src/main/java/com/google/cloud/genomics/utils/Contig.java

```diff
@@ -148,5 +184,78 @@ public Contig apply(String contigString) {
 
     return contigs;
   }
+  
+  /**
+   * Get a list of Contigs representing sharded windows of all genomic regions within the variant set.
```

Comment:
You're right, I was confused. This also explains why we spit out contigs that start at 0 - that's because it's what the API gives us. I still would like to see all those Genomic API-dependent features out of the contig class and into some helper class, but I'll grant you this is a matter of taste.